jangjuseong/controllers/postController.js

```diff
@@ -0,0 +1,68 @@
+const postService = require('../services/postService');
+
+const createPost = async (req, res) => {
+  try {
+    const { title, content, postImgUrl, userId } = req.body;
+
```

Comment:
postRouter 에서 validateToken 함수를 통해 토큰이 유효하고 유저 정보가 해당 controller 함수로 넘어오면 다음과 같이 작성할 수 있습니다.

```suggestion
const createPost = async (req, res) => {
  try {
    const { userId } = req.user; // next() 함수로 전달받은 req 객체에 user 정보가 추가된 새로운 req 객체
    const { title, content, postImgUrl, userId } = req.body;

```